Cargo.toml

```diff
@@ -219,8 +219,8 @@ imbl = "5.0.0"
 impl-tools = "0.10.3"
 iroh = { version = "=0.34.1", default-features = false }
 iroh-base = { version = "=0.34.1", default-features = false }
-iroh-next = { version = "0.35.0", default-features = false, package = "iroh" }
-iroh-next-base = { version = "0.35.0", default-features = false, package = "iroh-base" }
+iroh-next = { version = "=0.35.0", default-features = false, package = "iroh" }
```

Comment:
@maan2003 @elsirion I thought we don't need to pin this one, as this is the release we just want to keep rolling forward